.editorconfig

```diff
@@ -0,0 +1,30 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{py,rst,ini}]
+indent_style = space
+indent_size = 4
```

Comment:
why 4 for py, go and go but 2 for yaml/json? could you please add links to any official styling guidelines you followed?

Reply:
For py it's [pep-8](https://pep8.org/).
For yaml and json, it's just a matter of taste. It seems more concise for me.

Reply:
@surajkota

Reply:
Any updates? @surajkota